nltk/book.py

```diff
@@ -58,25 +64,142 @@ def texts():
     print("text8:", text8.name)
     print("text9:", text9.name)
 
+
 sent1 = ["Call", "me", "Ishmael", "."]
-sent2 = ["The", "family", "of", "Dashwood", "had", "long",
-         "been", "settled", "in", "Sussex", "."]
-sent3 = ["In", "the", "beginning", "God", "created", "the",
-         "heaven", "and", "the", "earth", "."]
-sent4 = ["Fellow", "-", "Citizens", "of", "the", "Senate",
-         "and", "of", "the", "House", "of", "Representatives", ":"]
-sent5 = ["I", "have", "a", "problem", "with", "people",
-         "PMing", "me", "to", "lol", "JOIN"]
-sent6 = ['SCENE', '1', ':', '[', 'wind', ']', '[', 'clop', 'clop',
-         'clop', ']', 'KING', 'ARTHUR', ':', 'Whoa', 'there', '!']
-sent7 = ["Pierre", "Vinken", ",", "61", "years", "old", ",",
-         "will", "join", "the", "board", "as", "a", "nonexecutive",
-         "director", "Nov.", "29", "."]
-sent8 = ['25', 'SEXY', 'MALE', ',', 'seeks', 'attrac', 'older',
-         'single', 'lady', ',', 'for', 'discreet', 'encounters', '.']
-sent9 = ["THE", "suburb", "of", "Saffron", "Park", "lay", "on", "the",
-         "sunset", "side", "of", "London", ",", "as", "red", "and",
-         "ragged", "as", "a", "cloud", "of", "sunset", "."]
+sent2 = [
+    "The",
+    "family",
+    "of",
+    "Dashwood",
+    "had",
+    "long",
+    "been",
+    "settled",
+    "in",
+    "Sussex",
+    ".",
+]
```

Comment:
This is not necessary esp. when it is clearer when the sentence tokens line max out the 80 chars as much as possible and then move to the next line.

Reply:
See my other comment about the philosophy of [black](https://www.reddit.com/r/Python/comments/8lmnzr/thoughts_on_black/).

If you prefer, existing formatting can be preserved with `# fmt: off` and `# fmt: on`

Reply:
I understand the non-compromising part but is that really what we want is a question to NLTK dev and community rather than just pushing the PR through.

Personally, going through each line to preserve the format with `# fmt: off` might be as good as enforcing the style before merging. But that's just my personal view. Maybe other users and devs have thoughts that differs.